Makefile

```diff
@@ -47,6 +48,40 @@ sgx: sdk sgx-env
 	cd sgx-root-enclave-bind && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build
 	cd veracruz-client && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build --lib --features sgx
 
+# TODO add -cli to other TEEs
+# TODO should we move all CLIs into the root dir?
+sgx-cli: sgx-env
+	# enclave binaries needed for veracruz-server
+	cd runtime-manager-bind && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build
+	cd sgx-root-enclave-bind && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build
+	# build CLIs in top-level crates
+	cd proxy-attestation-server && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build --features sgx --features cli
+	cd veracruz-server && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build --features sgx --features cli
+	cd veracruz-client && RUSTFLAGS=$(SGX_RUST_FLAG) cargo build --features sgx --features cli
```

Comment:
All of these sub-builds (`runtime-manager-bind`, `sgx-root-enclave-bind`, `proxy-attestation-server`, `veracruz-server`, `veracruz-client`) should probably have their own target in the Makefile (instead of all being under the same target) and be dependencies of the `sgx-cli` target

Reply:
I was trying to avoid refactoring the Makefile too much, but can make a general clean up pass if we want. Those would be good rules to have, though the combinatorial explosion of prefixes (sgx-root-enclave-bind, tz-root-enclave-bind, nitro-root-enclave-bind, etc) will be annoying.